docs/hadoop-catalog.md

```diff
@@ -25,19 +25,88 @@ Hadoop 3. If there's any compatibility issue, please create an [issue](https://g
+
+#### OSS fileset
+
+| Configuration item            | Description                                                                                                                                                                                                                   | Default value   | Required                   | Since version    |
+|-------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|----------------------------|------------------|
+| `filesystem-providers`        | The file system providers to add. Set it to `oss` if it's a oss fileset or a comma separated string that contains `oss` like `oss,gs,s3` to support multiple kind of fileset including `oss`.                                 | (none)          | Yes                        | 0.7.0-incubating |
+| `default-filesystem-provider` | The name default filesystem providers of this Hadoop catalog if users do not specify the scheme in the URI. Default value is `builtin-local`, for OSS, if we set this value, we can omit the prefix 'oss://' in the location. | `builtin-local` | No                         | 0.7.0-incubating |
+| `oss-endpoint`                | The endpoint of the Aliyun oss.                                                                                                                                                                                               | (none)          | Yes if it's a oss fileset. | 0.7.0-incubating |
+| `oss-access-key-id`           | The access key of the Aliyun oss.                                                                                                                                                                                             | (none)          | Yes if it's a oss fileset. | 0.7.0-incubating |
+| `oss-secret-access-key`       | The secret key of the Aliyun oss.                                                                                                                                                                                             | (none)          | Yes if it's a oss fileset. | 0.7.0-incubating |
+
+In the meantime, you need to place the corresponding bundle jar [gravitno-aliyun-bundle-{version}.jar](https://repo1.maven.org/maven2/org/apache/gravitino/aliyun-bundle/) in the Hadoop environment.
+
+
+:::note
+- Gravitino contains builtin file system providers for local file system(`builtin-local`) and HDFS(`builtin-hdfs`), that is to say if `filesystem-providers` is not set, Gravitino will still support local file system and HDFS. Apart from that, you can set the `filesystem-providerss` to support other file systems like S3, GCS, OSS or custom file system.
```

Comment:
You need to add a chapter to explain file system providers, I don't think users will know this without any explanation.

Reply:
Nevermind, I see you already explain this in the below.